wikimedia/wikipedia-ios
# Deep link wmf_languageVariantCode assignment fix

Fix based on discussion here: https://github.com/wikimedia/wikipedia-ios/pull/3894#discussion_r606277728

### Test Steps
1. Set an app language variant, say Traditional Chinese (HK).
2. Search for an article. Inspect Accept-Language header in mobile-html call and confirm it asks for `zh-hk`.
3. Choose language switcher from article and choose Chinese Simplified version. Inspect Accept-Language in mobile-html call and confirm it asks for `zh-hans`.
4. Search for a random article on desktop Zh wiki in Safari and deep link into the app.  Inspect Accept-Language in mobile-html call and confirm it asks for `zh-hk`.
5. Add Top Read widget to home screen.
6. Tap a top read row and deep link into the app. Inspect Accept-Language in mobile-html call and confirm it asks for `zh-hk`.
7. Go to app language, change primary language to Chinese Simplified
8. Go back to widget and tap row again to deep link into the app. Inspect Accept-Language in mobile-html call and confirm it asks for `zh-hans`.
